higher_health/templates/healthcheck_receipt.html

```diff
@@ -39,12 +38,12 @@ <h5 class="risk-cat__label">Risk Level: </h5>
               </p>
             </li>
             <li class="risk-cat__item">
-              <h5 class="risk-cat__label">Date Issues:</h5>
-              <p class="risk-cat__val">{% now "jS F Y H:i" %}</p>
+              <h5 class="risk-cat__label">Date Issued:</h5>
+              <p class="risk-cat__val">{{timestamp}}</p>
             </li>
             <li class="risk-cat__item">
               <h5 class="risk-cat__label">Expiry date:</h5>
-              <p class="risk-cat__val">-</p>
+              <p class="risk-cat__val">{% now "F jS Y" %}</p>
```

Comment:
Doesn't this need to be tomorrow?

Reply:
@miltontony Updated to today's 00:00 am static and unchanging time with a hint